src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.data;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public final class Constants {
+    public static final Logger logger = LogManager.getLogger("Errors");
```

Comment:
Please keep the Logger field per class. The Tag (what you have as "Errors") should be the name of the class. This lets you easily know where the error is coming from.